toolbox/special_resource_operator.py

```diff
@@ -29,6 +29,22 @@ def deploy_from_commit(git_repo, git_ref, image_tag=None):
 
         return PlaybookRun("sro_deploy_custom_commit", opts)
 
+    @staticmethod
+    def deploy_custom_image(custom_image, git_repo, git_ref):
+        """
+        Deploys the SRO operator from the given image.
+
+        Args:
+            custom_image: The custom image and tag to deploy from. E.g. image-registry/path/image:tag
```

Comment:
by reading the content of this PR, it's not really clear what this image is. 
I think it will be an image for the SRO operator/manager Pod, built by ART,
but I think you can clarify a bit here and there

Reply:
The commit looks different now, so I renamed it to be more clear.